Ocr-Yum-Mirror/scripts/install.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+#
+# LICENSE UPL 1.0
+#
+# Copyright (c) 1982-2018 Oracle and/or its affiliates. All rights reserved.
+#
+# Since: January, 2018
+# Author: gerald.venzl@oracle.com
```

Comment:
Perhaps should be updated but I'm fine also leaving it, given it's the same script.